File: challenges/codeeval/easy/067/Skhorn.pl
#!/usr/bin/perl

#use strict;
use warnings;

#Sample code to read in test cases:

open(INFILE, "<", $ARGV[0]) or die("Cannot open file $ARGV[0] for reading: $!");
while(my $line = <INFILE>) {
    next if($line =~ m/^\s$/);  # skip blank lines
    $line =~ s/(^\s|\s*$)//g;   # remove leading and trailing spaces
    my $hex = '0x'.$line;
    #print $hex;
    #print "\n";
    $hexval = hex($hex);
    print $hexval."\n";
    
}
close(INFILE);


#print "hi asdsad\n";

File: challenges/codeeval/easy/024/Skhorn.py
#!/usr/bin/python

import sys

class sum_int_files:

	sum_int = 0
	#file = open('test','r')
	with open(sys.argv[1], 'r') as test_cases:
		for line in test_cases:
		#for line in file:

			sum_int += int(line)

		print sum_int
	
x = sum_int_files()